pymc/tests/distributions/test_truncated.py

```diff
@@ -9,15 +9,15 @@
 from aesara.tensor.random.basic import GeometricRV, NormalRV
 
 from pymc import Censored, Model, draw, find_MAP, logp
-from pymc.distributions.continuous import Exponential, TruncatedNormalRV
+from pymc.distributions.continuous import Exponential, PyMCNormalRV, TruncatedNormalRV
 from pymc.distributions.shape_utils import change_dist_size
 from pymc.distributions.transforms import _default_transform
 from pymc.distributions.truncated import Truncated, TruncatedRV, _truncated
 from pymc.exceptions import TruncationError
 from pymc.tests.distributions.util import assert_moment_is_expected
 
 
-class IcdfNormalRV(NormalRV):
+class IcdfNormalRV(PyMCNormalRV):
```

Comment:
In that case let's use `PyMC` versions in all these test classes? Less confusing I think

Reply:
I added a comment to clarify why we are doing this then. Let me know if you think that this is good